imagedephi/redact/dicom.py

```diff
@@ -84,9 +86,20 @@ def __init__(self, image_path: Path, rules: DicomRules) -> None:
         self.metadata_redaction_steps = {}
         self.no_match_tags = []
         self.uid_map = {}
-        print(self.dicom_data)
 
         for element, _ in DicomRedactionPlan._iter_dicom_elements(self.dicom_data):
+            custom_metadata_key = "CustomMetadataItem"
+            if element.tag.group % 2 == 1:
```

Comment:
I'd add a comment here that odd number element groups are custom metadata, just so when we look back at this months from now we remember what is going on.